### DISCUSSION TITLE
Pb with Home Assistant 2024.10.1

### DISCUSSION THREAD

**Discussion by Typhus14** [joshuar/go-hass-agent#298] [General]
Hello,
I have install this new version and I meet errors with all sensors from go-hass-agent (in french, sorry): 
sensor.megalithe_uptime n'a plus de classe d'état
with this explication :
L'entité n'a plus de classe d'état
Nous avons généré des statistiques pour « megalithe Uptime » (sensor.megalithe_uptime) dans le passé, mais l'entité n'a plus de classe d'état, par conséquent, nous ne pouvons plus suivre ses statistiques à long terme.

Les statistiques ne peuvent pas être générées tant que cette entité n'a pas une classe d'état prise en charge.
Si la classe d'état était précédemment fournie par une intégration, il peut s'agir d'un bogue. Veuillez signaler un problème.
Si vous avez précédemment défini vous-même la classe d'état, veuillez la corriger. [Les différentes classes d'état et quand les utiliser peuvent être trouvées dans la documentation pour les développeurs.](https://developers.home-assistant.io/docs/core/entity/sensor/#long-term-statistics)
Si la classe d'état a été définitivement supprimée, vous pouvez supprimer de votre base de données les statistiques à long terme la concernant.
Souhaitez-vous supprimer définitivement les statistiques à long terme de sensor.megalithe_uptime de votre base de données ?

Thank you fort your help.

**Comment by joshuar** [comment]
Hey there, thanks for trying out Go Hass Agent!

No problem with French, translation tools are pretty good these days!

But apologies, rather than myself or a machine butchering French, I'll reply in English. I hope that is okay! 

This is interesting. The uptime sensor does set a state class and it hasn't changed since the previous version. Which version did you upgrade from?

Generally, this message is more of a warning. It may result in you being unable to view historical measurements of the uptime value, but otherwise should not affect the sensor in any other way. You can follow the instructions to delete the long-term statistics which will remove the message if you like.

**Comment by Typhus14** [comment]
Hello, 
Thanks for your answer. No problems with English, I'll try to reply without translation tools help (just a little ;-)).
Before I was using 2024.08 version (I installed go-hass-agent in this version). The Pb occur after the migration (I use Docker).
No problems for using your nice tool anyway, I just inform you of this new behavior (https://www.home-assistant.io/blog/2024/10/02/release-202410/#repairs-for-long-term-statistics).
Congratulations for your work.
